src/views/api-docs-view/components/path-truncation-aside/index.tsx

```diff
@@ -0,0 +1,37 @@
+import DevDotContent from 'components/dev-dot-content'
+import CodeBlock from '@hashicorp/react-code-block'
+
+/**
+ * Truncates HCP Packer API operation paths for clarity.
+ * Intended to be used with a `<PathTruncationAside />` in each operation intro.
+ */
+function truncatePackerOperationPath(path: string) {
+	return path.replace(
+		/\/packer\/\d\d\d\d-\d\d-\d\d\/organizations\/\{location.organization_id\}\/projects\/\{location\.project_id\}/,
+		''
+	)
+}
+
+/**
+ * Render an aside warning that operation paths have been truncated,
+ * and show the full path in a copy-able code block.
+ */
+function PathTruncationAside({ path }: { path: string }) {
+	return (
+		<>
+			{/*
+				@TODO replace DevDotContent & <div> with a base UI component
+				https://app.asana.com/0/1202097197789424/1203820006759167/f
+			*/}
+			<DevDotContent>
+				<div className="alert alert-info">
+					<strong>Note:</strong> Operation paths have been truncated for
+					clarity. The full path to this operation is:
```

Comment:
```suggestion
					clarity. The full path to this operation is below:
```
⛏️ Might make for _extra_ clarity to point to the codeblock below.